src/test/scala/scripts/s3wrapper/S3WrapperTest.scala

```diff
@@ -0,0 +1,74 @@
+/* package scripts.s3wrapper
```

Comment:
I forgot to uncomment this one

Reply:
the main problem is solved. thx. I will handle from now on